src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return (x) => {
+    return (y) => {
```

Comment:
it's not mistake but i think it will be better with more correctful names for x and y

Reply:
Here it's ok, cuz we just count sum